addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -197,6 +203,8 @@ options.registry.gallery = options.Class.extend({
             _.each(cols, function (col) {
                 var $col = $(col);
                 var height = $col.is(':empty') ? 0 : $col.find('img').last().offset().top + $col.find('img').last().height() - self.$target.offset().top;
+                // Neutralize invisible sub-pixel height differences.
+                height = Math.round(height);
```

Comment:
Can you explain what does it fix ? Maybe add screenshots in PR description :eyes:

Reply:
Screenshots will not show anything because it is a sub-pixel issue.
Using the default Image Wall images, the top one of the last column (writing on documents with a pen) is actually one pixel taller than the other ones. Once resized to the grid, they have the same height in number of pixels, but the column height with that image is 266.96... instead of 266.73... When putting the pictures in the column layout, each "next picture" is put inside the least tall column.
If you select the "writing" picture, and move it to the previous column, you will see that the 6th picture (the wine glass) moves together with it, because other columns are filled before that one by the previous pictures since they are less tall.
The fix makes the comparison happen on visible pixel differences.

Reply:
Ok I think this deserves it's own commit with your explanation and steps to reproduce :+1: